WcaOnRails/app/models/registration.rb

```diff
@@ -24,12 +27,30 @@ class Registration < ActiveRecord::Base
     end
   end
 
-  def pending?
-    accepted_at.nil?
+  def deleted?
+    !deleted_at.nil?
   end
 
   def accepted?
-    !pending?
+    !deleted? && !accepted_at.nil?
```

Comment:
Using `accepted_at` is causing problems since it is nil. So if we get `true && nil` it returns `nil` rather than `false`. So I might have to leave it as `!accepted_at.nil?`

Or is there a better way to write this?